src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationExecutor;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.impl.OperationExecutorImpl;
+import core.basesyntax.service.impl.ParserImpl;
+import core.basesyntax.service.impl.ReaderImpl;
+import core.basesyntax.service.impl.WriterImpl;
+import core.basesyntax.strategy.Operation;
+import core.basesyntax.strategy.impl.BalanceOperation;
+import core.basesyntax.strategy.impl.PurchaseOperation;
+import core.basesyntax.strategy.impl.ReturnOperation;
+import core.basesyntax.strategy.impl.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE = "src/main/resources/file.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/report.csv";
+
+    public static void main(String[] args) {
+        StorageDaoImpl storageDao = new StorageDaoImpl();
```

Comment:
Make your services independent and call them in the `main()` method. The `StorageDaoImpl` should not be instantiated directly in the `main()` method. Instead, consider using a factory or dependency injection to manage the creation of your service instances.